syz-fuzzer/proc.go

```diff
@@ -183,6 +184,15 @@ func (proc *Proc) triageInput(item *WorkTriage) {
 	}
 }
 
+func (proc *Proc) validateOutgoingInput(p *prog.Prog) {
+	for _, call := range p.Calls {
+		if call.Props.FailNth > 0 {
```

Comment:
Here duplicate one of the checks from checkProgram, but don't do the rest, so it's also inconsistent handling of different bad programs.
We could de-duplicate the checking logic, but I think we can simply remove the fuzzer check. Manager will check it right after receiving.

Reply:
Done.